tools/aws_lab_setup/roles/control_node/tasks/main.yml

```diff
@@ -22,13 +22,13 @@
     - control
 
 - name: Install Ansible
-  - pip:
```

Comment:
I'm confused by this part of the diff. I can't find this in the current master branch code.